### PR TITLE
Removing sudo from upstart script.

### PR DESCRIPTION
**Pull request by jlm3575**

### CODE REVIEW

File: templates/zookeeper.conf.j2 [hunk 1]
```diff
@@ -13,5 +13,7 @@ umask 007
 kill timeout 300
 
 script
-    exec sudo -u zookeeper -g zookeeper {{zookeeper_dir}}/bin/zkServer.sh start-foreground > /var/log/zookeeper/init-zookeeper.log 2>&1
+    # https://superuser.com/questions/213416/running-upstart-jobs-as-unprivileged-users
+    exec su -s /bin/sh -c 'exec "$0" "$@" &>>/var/log/zookeeper/init-zookeeper.log' zookeeper -- \
```
Comment (jlm3575) [inline templates/zookeeper.conf.j2]: A safer way to start the process for a different user.

**Comment by ernestas-poskus** [comment]
Hi please rebase on master, build is now fixed with https://github.com/AnsibleShipyard/ansible-zookeeper/pull/67

**Comment by jlm3575** [comment]
@ernestas-poskus rebased

**Comment by ernestas-poskus** [comment]
Thank you contribution

**Pull request merged**